docs/reference/executionPhases.md

```diff
@@ -46,6 +46,13 @@ you probably stopped execution during init phase.
 Continue execution and make sure you're in the exec phase before taking a look at data in your DataFrame.
 :::
 
+#### How does the Init Phase work for Spark Actions?
```

Comment:
@pgruetter I'm curious about your opinion on this new paragraph

Reply:
Thanks, explains it well 👍